From 8043471268e4256e4a7c08da032b7074777f3eb8 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Thu, 24 Sep 2015 21:49:44 +0900
Subject: [PATCH] Policy: [5.6.12] forbid colons in package version numbers.

Wording: Charles Plessy <plessy@debian.org>
Seconded: Didier 'OdyX' Raboud <odyx@debian.org>
Seconded: Sam Hartman <hartmans@debian.org>
Thanks: Jakub Wilk <jwilk@debian.org>
Closes: #792853
---
 debian/changelog         | 6 ++++++
 policy.sgml              | 6 ++----
 upgrading-checklist.sgml | 3 +++
 3 files changed, 11 insertions(+), 4 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index d3ddfe7..2696a48 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -39,6 +39,12 @@ debian-policy (3.9.7.0) unstable; urgency=low
     Wording: Charles Plessy <plessy@debian.org>
     Wording: Didier 'OdyX' Raboud <odyx@debian.org>
     Closes: #707851
+  * Policy: [5.6.12] forbid colons in package version numbers.
+    Wording: Charles Plessy <plessy@debian.org>
+    Seconded: Didier 'OdyX' Raboud <odyx@debian.org>
+    Seconded: Sam Hartman <hartmans@debian.org>
+    Thanks: Jakub Wilk <jwilk@debian.org>
+    Closes: #792853
 
  -- Bill Allombert <ballombe@debian.org>  Fri, 08 May 2015 15:10:02 +0200
 
diff --git a/policy.sgml b/policy.sgml
index 40f9f3f..7522f47 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -3224,11 +3224,9 @@ Package: libc6
 			Alphanumerics are <tt>A-Za-z0-9</tt> only.
 	          </footnote>
 	          and the characters <tt>.</tt> <tt>+</tt> <tt>-</tt>
-	          <tt>:</tt> <tt>~</tt> (full stop, plus, hyphen, colon,
+	          <tt>~</tt> (full stop, plus, hyphen,
 	          tilde) and should start with a digit.  If there is no
-	          <var>debian_revision</var> then hyphens are not allowed;
-	          if there is no <var>epoch</var> then colons are not
-	          allowed.
+	          <var>debian_revision</var> then hyphens are not allowed.
 		</p>
 	      </item>
 
diff --git a/upgrading-checklist.sgml b/upgrading-checklist.sgml
index 7db0197..f7c86af 100644
--- a/upgrading-checklist.sgml
+++ b/upgrading-checklist.sgml
@@ -40,6 +40,9 @@ picking your way through this list.
 Released xxx, 2015.
 
 </p><p><taglist>
+<tag>5.6.12</tag>
+  <item> Forbid colons in package version numbers.
+  </item>
 <tag>10.5</tag>
   <item> Symbolic links must not traverse above the root directory.
   </item>
-- 
2.1.4

